ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/Keystore/Util.hs

```diff
@@ -0,0 +1,15 @@
+module Ariadne.Wallet.Cardano.Kernel.Keystore.Util
+       ( esksToMap
+       , eskToKey
+       ) where
+
+import Pos.Core (AddressHash, addressHash)
+import Pos.Crypto (EncryptedSecretKey(..), PublicKey, encToPublic)
+
+import qualified Data.Map as Map (fromList)
+
+esksToMap :: [EncryptedSecretKey] -> Map (AddressHash PublicKey) EncryptedSecretKey
+esksToMap = Map.fromList . map (\encKey -> (eskToKey encKey, encKey))
+
+eskToKey :: EncryptedSecretKey -> AddressHash PublicKey
```

Comment:
Why is it called `eskToKey` if it returns hash?

Reply:
I think I adopted from previous code, I have changed.